.github/workflows/nodejs-next-test.js

```diff
@@ -0,0 +1,94 @@
+name: Node.js CI (Next)
+on:
+  push:
+    branches:
+      # Treat the below branches as special case for working on workflows
+      - actions-**
```

Comment:
```suggestion
      - actions-**
      - upcoming-**
```

This applies all branches that are like so: `upcoming-fix-something`, `upcoming-fix-something-else` created on the upstream.